sdk/go/x/auto/local_workspace.go

```diff
@@ -244,6 +245,37 @@ func (l *LocalWorkspace) RefreshConfig(ctx context.Context, fqsn string) (Config
 	return cfg, nil
 }
 
+// GetEnvVars returns the environment values scoped to the current workspace.
+func (l *LocalWorkspace) GetEnvVars() ([]string, error) {
+	if l.envvars == nil {
+		return nil, errors.New("unable to get environment values")
```

Comment:
I wonder if we could avoid having error return values for any of these functions? If we have an empty environment variable perhaps we can just return `[]string{}`

Reply:
Dropping the error works with me